editor/plugins/org.fusesource.ide.projecttemplates/pom.xml

```diff
@@ -1,16 +1,67 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-  <modelVersion>4.0.0</modelVersion>
-  
-  <parent>
-    <groupId>org.fusesource.ide.editor</groupId>
-    <artifactId>plugins</artifactId>
-    <version>8.0.0-SNAPSHOT</version>
-    <relativePath>..</relativePath>
-  </parent>
-  
-  <artifactId>org.fusesource.ide.projecttemplates</artifactId>
-  <packaging>eclipse-plugin</packaging>
-  <name>JBoss Fuse Tooling :: Camel Editor :: Plugins :: Project Templates</name>
+<project
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+	xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.fusesource.ide.editor</groupId>
+		<artifactId>plugins</artifactId>
+		<version>8.0.0-SNAPSHOT</version>
+		<relativePath>..</relativePath>
+	</parent>
+
+	<artifactId>org.fusesource.ide.projecttemplates</artifactId>
+	<packaging>eclipse-plugin</packaging>
+	<name>JBoss Fuse Tooling :: Camel Editor :: Plugins :: Project Templates</name>
+
+	<build>
+		<plugins>
+			<!-- Generate the template binaries -->
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-assembly-plugin</artifactId>
+				<configuration>
+					<finalName>template</finalName>
+					<descriptors>
+						<descriptor>.descriptors/blank-blueprint.xml</descriptor>
+						<descriptor>.descriptors/blank-java.xml</descriptor>
+						<descriptor>.descriptors/blank-spring.xml</descriptor>
+						<descriptor>.descriptors/simple-cbr-blueprint.xml</descriptor>
+						<descriptor>.descriptors/simple-cbr-java.xml</descriptor>
+						<descriptor>.descriptors/simple-cbr-spring.xml</descriptor>
+						<descriptor>.descriptors/simple-amq-blueprint.xml</descriptor>
+						<descriptor>.descriptors/simple-amq-spring.xml</descriptor>
+						<descriptor>.descriptors/medium-eap-wildfly-spring.xml</descriptor>
+						<descriptor>.descriptors/medium-cxf-codefirst-java.xml</descriptor>
+						<descriptor>.descriptors/medium-cxf-codefirst-spring.xml</descriptor>
+					</descriptors>
+					<outputDirectory>templates</outputDirectory>
+				</configuration>
+				<executions>
+					<execution>
+						<phase>compile</phase>

```

Comment:
shouldn't we bind it to generate-resources which is dedicated to "generate resources for inclusion in the package.?
https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html#Lifecycle_Reference

(even if i'm not sure that it makes a big differenc ein our case, perhaps just that we migth be able to call generate-resources phase only to go faster to regenerate the zip. which lead me think, what about a dedicated launch configuration for that?)